signer/src/storage/model.rs

```diff
@@ -145,10 +148,9 @@ impl From<bitcoin::Block> for BitcoinBlock {
 pub struct StacksBlock {
     /// Block hash.
     pub block_hash: StacksBlockHash,
-    /// Block height.
-    #[sqlx(try_from = "i64")]
+    /// Block height.    
     #[cfg_attr(feature = "testing", dummy(faker = "0..u32::MAX as u64"))]
-    pub block_height: u64,
+    pub block_height: StacksBlockHeight,
```

Comment:
qq: why do we still need the `cfg_attr` for this (and other) while we removed it in `BitcoinBlock::block_height`?

Reply:
Yea, need to remove this, thanks

Reply:
[5e704e6](https://github.com/stacks-sbtc/sbtc/pull/1448/commits/5e704e6bc9abd974bac92c104945db7f487ac2f4)